pkgs/tools/X11/alttab/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, pkgconfig, ronn, libpng, uthash
+, xorg }:
+
+stdenv.mkDerivation rec {
+  version = "1.4.0";
+
+  name = "alttab-${version}";
+
+  src = fetchFromGitHub {
+    owner = "sagb";
+    repo = name;
```

Comment:
This doesn't build for me, I think the version should not be in the repo name. Changing to `repo = "alttab"` makes this work for me

Reply:
Ouch... Thanks for catching this! I changed the `name` to its current definition at some point... Apparently this didn't tried to fetch from GitHub again.

If I understand the `pname` mechanics right, it should be enough to set `pname = "alttab"` above and set `repo = pname` instead, right?

Reply:
that should work